src/shared.ts

```diff
@@ -20,6 +20,9 @@ export class MessageAction implements vscode.MessageItem {
     constructor(public title: string, public command?: string, public description?: string, public callback?: Function, public args?: any[]) { }
 }
 
+// Build
+export const OUTPUT_FOLDER_NAME = '_site';
```

Comment:
Add DEFAULT as prefix?

Reply:
it is not customizable, it is a const, so `Default` prefix is not necessery

---
In reply to: [365630823](https://github.com/docascode/vscode-docs-build/pull/13#discussion_r365630823) [](ancestors = 365630823)